Task1: #cone_track parameter
    voxel_size: 0.05

    min_x_: 0.0
    max_x_: 2.0

    min_y_: -0.5
    max_y_: 0.5

    tolerance: 0.05
    cluster_size_min: 1
    cluster_size_max: 30

    cone_dist: 0.3

    #왼쪽 혹은 오른쪽이 부족하여 채워주려고 하는데 y값을 얼마나 더할거냥
    dist_offset: 0.5

    LD: 0.6


Task2: #tunnel parameter
    voxel_size: 0.05

    min_x_: 0.0
    max_x_: 1.0

    min_y_: -1.5
    max_y_: 1.5

    cone_dist_: 0.2
    chan_dist_offset: 0.35
    dist_offset: 1.3
    outcourse_offset: 0.0
    straight_count_param: 18
    straight_angle: -8
    ratio_line: 0.15
    ratio_line2: 0.35 # 탈출할 때 직진 조향을 위한 flag
    done_count_param: 5
    
    corner_speed: 0.3
    LD: 0.6

Task3: #static parameter
    voxel_size: 0.05

    min_x_: 0.0
    max_x_: 5.0

    min_y_: -0.5
    max_y_: 0.5

    tolerance: 0.3

    cluster_size_min: 1
    cluster_size_max: 500

Task4: #static parameter
    voxel_size: 0.01

    min_x_: 0.0
    max_x_: 5.0

    min_y_: -0.5
    max_y_: 0.5

    tolerance: 0.3

    cluster_size_min: 1
    cluster_size_max: 500

Task5: #static parameter
    voxel_size: 0.01

    min_x_: 0.0
    max_x_: 5.0

    min_y_: -0.5
    max_y_: 0.5

    tolerance: 0.3

    cluster_size_min: 1
    cluster_size_max: 500